7.Product_Designer-FlexBox/css/style.css

```diff
@@ -0,0 +1,148 @@
+/* CSS reset */
+*{
+    margin:0;
+    padding:0;
+
+    box-sizing: border-box;
+    overflow:auto;
+}
+body{
+    background-color: #000;
+    color: #fff;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+
+/* ================================TITLE CONTENT=================================== */
+.designer{
+    display: flex;
+    flex-direction: column;
+    row-gap: 130px;
+    padding: 40px 172px 0px;
+}
+/* Designer Mail */
+.designer-mail{
+    display: flex;
+    justify-content: flex-end;
+    font-size: 24px;
+}
+/* Designer Name*/
+.designer-name{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    row-gap: 6px;
+}
+.designer-name h1{
+    font-weight: 300;
+    font-size: 150px;
+}
+.bold{
+    font-weight: 900;
+}
+.designer-name h3{
+    font-size: 50px;
+    color: #cecece;
+}
+.blue{
+    color: #4990e2;
+}
+/* =================================GALLERY CONTENT================================ */
+.gallery{
+    margin-top:100px;
+    padding-right: 50px;
+    padding-left: 50px;
+    display:flex;
+    flex-wrap: wrap; 
+    row-gap: 20px;
+    column-gap: 20px;
+    justify-content: center;
+}
+.gallery-image{
+    width: 400px;
+}
+.gallery-image img{
+    width: 100%;
+}
+
+/* =================================BRANDS CONTENT================================= */
+.brands-content{
+    display:flex;
+    flex-direction: column;
+    row-gap:100px;
+    margin-top: 175px;
+}
+/* ----------Brands Heading--------------- */
+.brand-heading{
+    display:flex;
+    justify-content: center;
+}
+.brand-heading h2{
+    font-size: 50px;
+    color: #cecece;
+}
+.green{
+    color: #87d979;
+}
+/* --------Brands Logos Wrapper----------- */
+.brands-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    column-gap:100px;
+}
+.brand-logo{
+    display: flex;
+    justify-content: center;
+    width: 120px;
+}
+/* ===============================FOOTER CONTENT=================================== */
+footer{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: center;
+    row-gap:120px;
+    margin: 280px auto 170px;
+}
+/* -----------Designer Border---------- */
+.divider{
+    display: flex;
+    justify-content:space-between;
+    padding-left: 720px;
+    padding-right: 720px;
+}
+.small-divider{
+    width: 12%;
+}
+.large-divider{
+    width: 76%;
+}
```

Comment:
I think you can avoid padding here, and with 'justify-content: center' and margin right for small-divider will do the trick.
Reason: Padding won't be consistent for all the viewports (1728x1080, 1920x1080, 2160x1440)